php.php

```diff
@@ -0,0 +1,18 @@
+<?php
+    class Deposit {
+
+        /**
```

Comment:
Typo inside comment in token 'ownner'. Possible corrections:owner (76),owned (1),owners (0)